kafka-sample/c#/README.md

```diff
@@ -17,46 +18,33 @@ Package Links:
 
 Refer to [c#.proj](c%23.csproj) for version details and if you want to update versions.
 
-On linux systems, the client expects the hosts' ca-certificate at `/etc/pki/tls/ca-bundle.crt` (fedora/redhat defaults).
-If your systems default location differs from that (e.g. Debian/Ubuntu), please make sure to copy or symlink the
-certificate.
+Our Kafka server uses Let’s Encrypt CA for SSL/TLS certificates. Confluent.Kafka library use the operating system's 
+default trusted root CA certificates for secure connections. Please ensure that your system has the Let's Encrypt root 
+certificates installed. These certificates are usually included in the system's trusted root store by default. If not, 
+please install them manually.
 
-Example for Debian/Ubuntu distributions:
+For Debian/Ubuntu distributions, the system CA certificates can be updated as:
 ```bash
-mkdir -p /etc/pki/tls/certs
-ln -s /etc/ssl/certs/ca-certificates.crt /etc/pki/tls/certs/ca-bundle.crt
+sudo update-ca-certificates
 ```
 
 How to use
 ----------
 
-In order to use the sample please change the following parameters. Those parameters should previously send to you.
+In order to use the sample, please substitute the following parameters with those you have received via secure email.
 
 ```cs
-  var clientId = "YOUR_CLIENT_ID";                    // use the client you have received
-  var clientSecret = "YOUR_CLIENT_SECRET";            // use the secret you have received
-  var topic = $"vehiclesignals.{clientId}";           // use topic for the client you have received
-  var consumerGroup = $"{clientId}.GROUP_ID_POSTFIX"; // you can change the postfix of your consumer group
-
-  var bootstrapUrl = "BOOTSTRAP_URL";                 // use the correct broker url for your region
-  var oauthTokenApiUrl = "OAUTH_TOKEN_API_URL";       // use the correct token API url for your region
-```
-
-You also need to provide the CA certificate location.
-
-```cs
-  var rootCaFile = @"cluster-ca.crt";                 // file path of your CA certificate (must be a PEM file)
+  var clientName = "YOUR_CLIENT_NAME";                  // use the client name you have received 
+  var clientId = "YOUR_CLIENT_ID";                      // use the client id you have received
+  var clientSecret = "YOUR_CLIENT_SECRET";              // use the secret you have received
+  var scope = "SCOPE";                                  // use the correct scope for your region
+  var topic = $"vehiclesignals.{clientName}";           // use topic for the client you have received
+  var consumerGroup = $"{clientName}.GROUP_ID_POSTFIX"; // you can change the postfix of your consumer group
```

Comment:
I guess this is MBCon specific. We should not treat it as a general approach.

Reply:
Implemented as suggested.